### PR TITLE
bump to v.0.20

### PR DESCRIPTION
**Pull request by carlioth**
Cleaner config now

### CODE REVIEW

File: ingress/ingress-nginx-dep.yaml [hunk 1]
```diff
@@ -37,16 +47,11 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: metadata.namespace
-          resources:
```
Comment (FredrikFolkesson) [inline ingress/ingress-nginx-dep.yaml]: Why remove the resource requirement?

File: ingress/ingress-nginx-dep.yaml [hunk 2]
```diff
@@ -2,32 +2,42 @@ apiVersion: extensions/v1beta1
...
+          image: quay.io/kubernetes-ingress-controller/nginx-ingress-controller:0.20.0
           args:
             - /nginx-ingress-controller
-            - --default-backend-service=$(POD_NAMESPACE)/default-http-backend
             - --configmap=$(POD_NAMESPACE)/nginx-configuration
-            - --tcp-services-configmap=$(POD_NAMESPACE)/tcp-services
-            - --udp-services-configmap=$(POD_NAMESPACE)/udp-services
             - --publish-service=$(POD_NAMESPACE)/ingress-nginx
             - --annotations-prefix=nginx.ingress.kubernetes.io
+          securityContext:
```
Comment (FredrikFolkesson) [inline ingress/ingress-nginx-dep.yaml]: What is this?

File: ingress/ingress-nginx-cfg.yaml [hunk 3]
```diff
@@ -61,16 +61,5 @@ metadata:
...
-metadata:
-  name: tcp-services
-  namespace: ingress-nginx
----
-kind: ConfigMap
-apiVersion: v1
-metadata:
-  name: udp-services
-  namespace: ingress-nginx
+    app.kubernetes.io/name: ingress-nginx
```
Comment (FredrikFolkesson) [inline ingress/ingress-nginx-cfg.yaml]: How does these labels work?

File: ingress/ingress-nginx-dep.yaml (hunk 2 above)
Comment (carlioth) [reply ingress/ingress-nginx-dep.yaml]: Previously it's been running as root, now it's limited

File: ingress/ingress-nginx-dep.yaml (hunk 1 above)
Comment (carlioth) [reply ingress/ingress-nginx-dep.yaml]: we should keep that! I've lowered it now thou, looking into our history it never moves above 60 MB

File: ingress/ingress-nginx-dep.yaml [hunk 4]
```diff
@@ -39,14 +49,14 @@ spec:
...
+                memory: "128Mi"
               limits:
-                memory: "256Mi"
+                memory: "128Mi"
           ports:
-          - name: http
-            containerPort: 80
-          - name: https
-            containerPort: 443
+            - name: http
```
Comment (wennmo) [inline ingress/ingress-nginx-dep.yaml]: Should this be limited to https only?

File: ingress/ingress-nginx-service.yaml [hunk 5]
```diff
@@ -1,22 +1,21 @@
...
+    app.kubernetes.io/name: ingress-nginx
+    app.kubernetes.io/part-of: ingress-nginx
   ports:
-  - name: http
-    port: 80
-    targetPort: http
-  - name: https
-    port: 443
-    targetPort: https
+    - name: http
```
Comment (wennmo) [inline ingress/ingress-nginx-service.yaml]: Same here, do we need to expose both http and https?

File: ingress/ingress-nginx-dep.yaml (hunk 4 above)
Comment (carlioth) [reply ingress/ingress-nginx-dep.yaml]: I've added an issue to discuss this. https://github.com/qlik-oss/core-scaling/issues/89

**Pull request merged**